app/_src/guides/restrict-permissions-to-selected-namespaces-on-kubernetes.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Restrict permissions to selected namespaces on Kubernetes
+content_type: tutorial
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+{% assign kuma-demo = kuma | append: "-demo" %}
+{% assign kuma-another-demo = kuma | append: "-another-demo" %}
+
+By default, {{ Kuma }} deployed on Kubernetes has permission to observe and react to events from resources across the entire cluster. While this behavior simplifies initial setup and testing, it might be too permissive for production environments. Limiting {{ Kuma }}'s access to only necessary namespaces helps enhance security and prevents potential impact on unrelated applications.
+
+Starting from version 2.11, you can restrict the permissions granted to the {{ Kuma }} control plane. This guide explains how to limit {{ Kuma }} to specific namespaces, giving you greater control over security and resource management.
+
+## Prerequisites
+
+Before you start, ensure you have:
+
+* A clean, running, and accessible Kubernetes cluster
+* The `kubectl` command-line tool installed and configured
+* The `kumactl` command-line tool installed
+
```

Comment:
Usually when you get into list items that go higher than 5 it indicates you are going to loose the reader.

Here I suggest a way to reorganise this in subsections where each section is a logical end.
What you get from these sections is that they have each a few items which makes it easier to track and makes the tuto look less like an insurance contract.

Reply:
Do you mean amount of steps in this case?

Reply:
Yes create sub titles with at most 5 items in each.